app/trip_price_count.py

```diff
@@ -0,0 +1,47 @@
+import math
+from datetime import datetime
+from app.customer import Customer
+from app.shop import Shop
+
+
+def count_trip_distance(client: Customer,
+                        shop: Shop,
+                        fuel_price: float) -> float:
+    trip_road_distance = math.dist(client.location,
+                                   shop.location) * 2
+    one_km_cost = client.car.fuel_consumption * fuel_price / 100
+    trip_price = trip_road_distance * one_km_cost
+    return round(trip_price, 2)
+
+
+def goods_price_counting(get_client: Customer, get_shop: Shop) -> float:
+    total = []
+    for product_name in get_client.product_cart:
+        if product_name in get_shop.products:
+            product_client = get_client.product_cart.get(product_name)
+            product_shop = get_shop.products.get(product_name)
+            total.append(product_client * product_shop)
+    return sum(total)
+
+
+def shop_bill(shop_client: Customer,
+              shop: str,
+              shops: list
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]